tests/createWallets.spec.ts

```diff
@@ -0,0 +1,198 @@
+import { test } from "@playwright/test";
+import puppeteer from "puppeteer";
+import { getDocument, queries, wait } from "pptr-testing-library";
+const { getByText, getByLabelText } = queries;
+import { USER } from "complete-randomer";
+
+const delay = async (time) => {
+  return new Promise(function (resolve) {
+    setTimeout(resolve, time);
+  });
+};
+
+const loadExtension = async () => {
+  const extensionID = "lmcohpmphdnfjpoejmnpeffoffngmkoi";
+  const extensionPath = "./dist/development/chrome";
+  const extensionOptionHtml = "welcome.html";
+  const extPage = `chrome-extension://${extensionID}/${extensionOptionHtml}`;
+  const browser = await puppeteer.launch({
+    headless: false,
+    args: [
+      `--disable-extensions-except=${extensionPath}`,
+      `--load-extension=${extensionPath}`,
+      "--disable-gpu",
+      "--disable-dev-shm-usage",
+      "--no-sandbox",
+      "--disable-setuid-sandbox",
+    ],
+  });
+
+  // trick to bring the new welcome page to the front
+  await delay(1000);
+  const page = await browser.newPage();
+  await page.bringToFront();
+  await page.setViewport({ width: 1366, height: 768 });
+  await page.goto(extPage);
+
+  return { page, browser };
+};
+
+test.describe("Create or connect wallets", () => {
+  test("successfully creates an Alby wallet", async () => {
+    const user = USER.SINGLE();
```

Comment:
```suggestion
    process.env['WALLET_CREATE_URL'] = 'https://app.regtest.getalby.com/api/users';
    const user = USER.SINGLE();
```

Reply:
setting the environment variable should probably not happen in the test case because this has global side effects, and is not only changing the env variable for the specific test case. we should set this globally for test runs.

Reply:
So we better set this in the `loadExtension` function?

Reply:
Setting it in the Github pipeline is not a great idea because someone that runs the tests locally should also connect to the regtest environment when creating wallets. Another option is to have the default value _in the code_ point to the regtest URL, and override this somehow in the main process with the production host.

Reply:
+1 for regest defaults. we overwrite env variables currently already for production builds.
can you update it?

Reply:
I don't know the `loadExtension` function. but if you set a global variable in a test it has an effect on all the other tests. - which is unintended. 
tests run (hopefully) in random order thus this will also only sometimes influence a test.

Reply:
How can I add a commit to this PR since the base is coming from MiguelMedeiros' fork?

Reply:
Or should I check this out in a new branch and make a new PR?

Reply:
you should be able to add commits to this PR (this branch of his fork) GH allows that for maintainers.

Reply:
I'm not allowed to do that.
```
To github.com:MiguelMedeiros/lightning-browser-extension.git
 ! [remote rejected]   MiguelMedeiros/test/integration-tests -> 
MiguelMedeiros/test/integration-tests (permission denied)
error: failed to push some refs to 'github.com:MiguelMedeiros/lightning-browser-extension.git'
```

Reply:
interesting. actually that's default normally. I also use the `gh` CLI to checkout and push.

Reply:
Just pushed a docs change to test this. Works for me. I'm not using the Github CLI. Just plain git.  
I cloned his repo, checked out the branch, created a commit and pushed. 🤷‍♂️